tests/gis_tests/rasterapp/test_rasterfield.py

```diff
@@ -356,3 +358,9 @@ def test_db_function_errors(self):
         msg = "Distance function requires a GeometryField in position 1, got RasterField."
         with self.assertRaisesMessage(TypeError, msg):
             RasterModel.objects.annotate(distance_from_point=Distance("rastprojected", point)).count()
+
+    def test_lhs_with_index_rhs_without_index(self):
+        with CaptureQueriesContext(connection) as queries:
+            RasterModel.objects.filter(rast__0__contains=json.loads(JSON_RASTER)).exists()
+
```

Comment:
Might be nice to replace this blank line with a comment that explains why the regex verification is needed.